tokio/src/sync/mpsc/chan.rs

```diff
@@ -488,6 +515,10 @@ impl Semaphore for AtomicUsize {
         }
     }
 
+    fn is_closed(&self) -> bool {
```

Comment:
Do we need this implementation?

Reply:
Do you mean unbounded channels should not have `fn is_closed`?